Src/.editorconfig

```diff
@@ -213,6 +213,36 @@ dotnet_diagnostic.CS1591.severity = silent
 dotnet_diagnostic.CS1712.severity = silent
 dotnet_diagnostic.CS1734.severity = silent
 
+# CS0649: Field is never assigned to, and will always have its default value
```

Comment:
likely not the same, fyi I got from Michael before that comments should end with a `.`